incubator/mariadb-cluster/templates/master-deploy.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: {{ template "master.fullname" . }}
   labels:
     app: "{{ template "name" . }}"
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    chart: {{ template "mariadb.chart" . }}
     component: "master"
     release: "{{ .Release.Name }}"
```

Comment:
You can use instead:

```
{{ .Release.Name | quote }}
```

This apply to every line with format: `""{{ ... }}"`